var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -88,7 +88,6 @@ class Llvm(CMakePackage, CudaPackage):
     variant(
         "flang",
         default=False,
-        when="@11: +clang",
```

Comment:
Why is this being removed?

Reply:
https://github.com/spack/spack/pull/32476#issuecomment-1235585173